server/svix-server/Cargo.toml

```diff
@@ -56,6 +56,7 @@ thiserror = "1.0.30"
 bytes = "1.1.0"
 blake2 = "0.10.4"
 chacha20poly1305 = "0.9.0"
+getrandom = "0.2.4"
```

Comment:
Open source review (even if it's already a dep, you can then just say "we already use it").